backend/src/main/java/com/bounswe2024group10/Tradeverse/dto/portfolio/CreatePortfolioRequest.java

```diff
@@ -0,0 +1,31 @@
+package com.bounswe2024group10.Tradeverse.dto.portfolio;
+
+public class CreatePortfolioRequest {
```

Comment:
Portfolio visibility is not an attribute of portfolio, we are storing it in User class so you can remove it

Reply:
removed.